screens/TaskDetailsScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+
+// Define o componente TaskDetailsScreen usando TypeScript para tipagem das props
+const TaskDetailsScreen: React.FC<{ route: any }> = ({ route }) => {
+  // Extrai a tarefa passada como parâmetro através da navegação
+  const { task } = route.params;
+  // Converte a string de data armazenada em um objeto Date para usar métodos de data
+  const createdAtDate = new Date(task.createdAt);
+
+  // Renderiza o componente na tela
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>Nome da Tarefa: {task.name}</Text> 
+      <Text>Data Criação: {createdAtDate.toLocaleDateString('pt-BR')}</Text>
```

Comment:
Eu removeria o `pt-BR` apenas para garantir que independente do idioma que for interpretado, mostrar a data no formato que foi interpretado